bin/deploy.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env -S node --require ts-node/register
+
+const contracts = require('../src/index.ts');
+const { providers, Wallet } = require('ethers')
+const { JsonRpcProvider } = providers;
+
+const key = process.env.DEPLOYER_PRIVATE_KEY;
+const sequencerKey = process.env.SEQUENCER_PRIVATE_KEY;
+const web3Url = process.env.L1_NODE_WEB3_URL || 'http://127.0.0.1:8545';
+const MIN_TRANSACTION_GAS_LIMIT = process.env.MIN_TRANSACTION_GAS_LIMIT || 0;
+const MAX_TRANSACTION_GAS_LIMIT = process.env.MAX_TRANSACTION_GAS_LIMIT || 1000000000;
+const MAX_GAS_PER_QUEUE_PER_EPOCH = process.env.MAX_GAS_PER_QUEUE_PER_EPOCH || 1000000000000;
```

Comment:
Curious if @ben-chain has input here but my guess is that this default is a bit too high when coupled with the `600` second epoch time. I think a safer one is `500000000` , but maybe even safer is `250000000`

Reply:
> 250000000

This is the one to use